src/decorator/relations/OneToMany.ts

```diff
@@ -2,14 +2,15 @@ import {getMetadataArgsStorage, ObjectType, RelationOptions} from "../../";
 import {RelationMetadataArgs} from "../../metadata-args/RelationMetadataArgs";
 
 /**
- * One-to-many relation allows to create type of relation when Entity1 can have multiple instances of Entity2.
- * Entity2 have only one Entity1. Entity2 is an owner of the relationship, and storages Entity1 id on its own side.
+ * A one-to-many relation allows creating the type of relation where Entity1 can have multiple instances of Entity2,
+ * but Entity2 has only one Entity1. Entity2 is the owner of the relationship, and stores the id of Entity1 on its
+ * side of the relation.
  */
 export function OneToMany<T>(typeFunctionOrTarget: string|((type?: any) => ObjectType<T>), inverseSide: string|((object: T) => any), options?: RelationOptions): PropertyDecorator {
     return function (object: Object, propertyName: string) {
         if (!options) options = {} as RelationOptions;
 
-        // now try to determine it its lazy relation
+        // Now try to determine if it is a lazy relation.
```

Comment:
there is no need to upper-case inline comment and put a dot to the end. This is a default code style in all TypeORM source files.